connectors-home/modules/ROOT/pages/streaming-strategies.adoc

```diff
@@ -0,0 +1,41 @@
+= Streaming Strategies
+
```

Comment:
What is this file for? Is it a partial or an actual .adoc that should be specified in the nav?

Reply:
No, it shouldn't show up in the output. It's a partial. I probably have it in the wrong folder.